index.d.ts

```diff
@@ -8,6 +8,7 @@ declare namespace cleanStack {
 		@default false
 		*/
 		readonly pretty?: boolean;
+		readonly basePath?: string;
```

Comment:
index.d.ts and readme should be in sync documentation-wise. This needs a doc comment.

Reply:
doc comment added in https://github.com/sindresorhus/clean-stack/pull/18/commits/0bdea8265bd95a58286545414df6d0d1f2e9c540.

Is the `@default undefined` neccessary, by the way?

Reply:
> Is the @default undefined neccessary, by the way?

No

Reply:
removed